spec/features/result_filters/vacancy_spec.rb

```diff
@@ -6,14 +6,25 @@
   let(:courses_url) do
     "http://localhost:3001/api/v3/recruitment_cycles/2020/courses"
   end
+
   let(:subjects_url) do
-    "http://localhost:3001/api/v3/subject_areas?include=subjects"
+    "http://localhost:3001/api/v3/subjects?fields%5Bsubjects%5D=subject_name,subject_code&sort=subject_name"
   end
 
   let(:base_parameters) { results_page_parameters }
 
+  let(:stub_subjects_request) do
+    stub_request(
+      :get,
+      "http://localhost:3001/api/v3/subjects?fields%5Bsubjects%5D=subject_name,subject_code&sort=subject_name",
```

Comment:
We should probably use `subjects_url` here or remove lines 10-12 since they don't seem to be used.

```suggestion
      subjects_url,
```

Reply:
Thanks @AgaDufrat. I've removed this and added the helper as per your other suggestion.